Components/Optimizers/AdaptiveStochasticVarianceReducedGradient/itkStochasticVarianceReducedGradientDescentOptimizer.cxx

```diff
@@ -289,7 +289,7 @@ StochasticVarianceReducedGradientDescentOptimizer::AdvanceOneStep()
     temp.t_Optimizer = this;
 
     /** Call multi-threaded AdvanceOneStep(). */
-    auto local_threader = ThreaderType::New();
+    auto local_threader = MultiThreaderBase::New();
```

Comment:
Might use thread pools, so might improve the performance of StochasticVarianceReducedGradientDescentOptimizer 👍